README.md

```diff
@@ -31,6 +31,11 @@ Install all the dependencies with the below. This is a one-time activity (for an
 pip install -r requirements.txt
 ```
 
+If you're on Windows, install dependencies with below command
+```
+python3 -m pip install -r requirements.txt
```

Comment:
Sometimes it gives syntaxerror which can be resolved by using python3

Reply:
It's not because of syntax error, it's because you have more python version installed in your machine and set in path or your executable named is changed to python3.
It's Better to use virtual environment in such case.

`python3 -m pip install -r requirements.txt` may not be correct to use if you don't have executable python3 which does not comes with default installs of python usually.

Reply:
yeah. and for pip also in windows not default.
I am novice for windows(usually use Linux only) but this is how I find this way to execute pip in windows system